onnx/helper.py

```diff
@@ -52,6 +52,10 @@ def process(release_version: Text, ir_version: int, *args: Any) -> None:
         for pair in zip(['ai.onnx', 'ai.onnx.ml', 'ai.onnx.training'], args):
             if (pair not in result):
                 result[pair] = ir_version
+                if pair[0] == 'ai.onnx':
+                    result[('', pair[1])] = ir_version
+                if pair[0] == 'ai.onnx.training':
+                    result['ai.onnx.preview.training', pair[1]] = ir_version
```

Comment:
Some training testing models originally used ai.onnx.preview.training and didn't have IR_VERSION. In this PR, these models are updated with specified IR_VERSION.